src/SFA.DAS.EmployerApprenticeshipsService.Domain.UnitTests/Data/Entities/AccountTests/WhenISendTransferConnection.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EAS.Domain.Data.Entities.Account;
+using SFA.DAS.EAS.Domain.Models.TransferConnections;
+using SFA.DAS.EAS.Domain.Models.UserProfile;
+
+namespace SFA.DAS.EmployerApprenticeshipsService.Domain.UnitTests.Data.Entities.AccountTests
+{
+    [TestFixture]
+    public class WhenISendTransferConnection
```

Comment:
The tests could be added to the `SFA.DAS.EAS.Application.UnitTests.Commands.SendTransferConnectionInvitation.WhenISendATransferConnectionInvitation` class or we could decide not to test our command handlers since they'll be fairly thing going forward.

Reply:
I think you're suggesting testing the behaviour indirectly by testing the command? The code under test is the Account entity because it implements the behaviour that is being tested. This seems direct - I'm probably missing something so will run this by you tomorrow.

Reply:
'tis done